packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -945,6 +945,48 @@ void main() {
       expect(material.color, equals(Colors.red));
     });
 
+    testWidgets('make clipBehavior be set correctly for MenuAnchor', (WidgetTester tester) async {
```

Comment:
```suggestion
    testWidgets('MenuAnchor clip behavior', (WidgetTester tester) async {
```
Simply the title and make it similar to other clip behavior in the framework.

Reply:
Done